### PR TITLE
rss: add GitHub

### PR DESCRIPTION
**Pull request by dragon-yuan**
Sorry, tried too many times before.
Thanks for the correction of some friends.
Add rss for GitHub, and add access_token now.

Header(https://developer.github.com/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/):
`
X-Accepted-OAuth-Scopes →
X-OAuth-Scopes →
X-RateLimit-Limit →5000
X-RateLimit-Remaining →4990
`

### CODE REVIEW

File: routes/github/repos.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const axios = require('../../utils/axios');
+const config = require('../../config');
+
+module.exports = async (ctx) => {
+    const token = 'ac97f3af103c6e26ed8d8ac1606af5f46398fee4';
```
Comment (stu43005) [inline routes/github/repos.js]: move to [config.js](https://github.com/DIYgod/RSSHub/blob/master/config.js)?

Comment (dragon-yuan) [reply routes/github/repos.js]: OK！It's great. Thanks!

Comment (tgly307) [reply routes/github/repos.js]: 记得把自己的token去掉哟，改成环境变量就好。

Comment (dragon-yuan) [reply routes/github/repos.js]: @tgly307 正在修改，学习到了很多思想，谢谢！

**Pull request merged**